java/maven.grammar/src/org/netbeans/modules/maven/codegen/ExclusionGenerator.java

```diff
@@ -90,9 +90,9 @@ public int write() {
                                 // now check the active profiles for the dependency..
                                 List<String> profileNames = new ArrayList<String>();
                                 NbMavenProject project = prj.getLookup().lookup(NbMavenProject.class);
-                                Iterator it = project.getMavenProject().getActiveProfiles().iterator();
+                                Iterator<org.apache.maven.model.Profile> it = project.getMavenProject().getActiveProfiles().iterator();
```

Comment:
Why not import this class?

Reply:
Hey @hectorespert , I wish I could do this.. But, there is already another Profile class being used, org.netbeans.modules.maven.model.pom.Profile. And it's not easy to distinguish between the two..

So we're going to have to leave this one alone..

Thanks for the feedback and hopefully you can now approve it for 12.1..